lib/nationsync/cli.rb

```diff
@@ -30,19 +30,22 @@ def save_config!
       File.open(fn, 'w') { |f| YAML.dump(@config, f) }
     end
   end#/no_commands
-  
+
   desc "init", "Initialize a theme in this directory"
   def init
     puts "Creating .nbconfig1"
     domain   = HL.ask("Domain: ").to_s
     email    = HL.ask("Email: ").to_s
     password = HL.ask("Password: ") {|q| q.echo = false }.to_s
-    
-    print "Fetching credentials..."
+
+    print "Fetching credentials...\n"
```

Comment:
The original intent with a `print` like this is to get an output in the console like this:

```
Fetching credentials... Done
```

However, given that we may be logging stuff in between it's probably best to switch to `puts`:

```ruby
puts "Fetching credentials for #{domain}"

auth = NationSync::Auth.new(domain, email, password)
success = auth.authenticate!
return false unless success

puts "Successfully fetched credentials"
```